src/pyodide/internal/pool/builtin_wrappers.ts

```diff
@@ -1,6 +1,7 @@
 import type { getRandomValues as getRandomValuesType } from 'pyodide-internal:topLevelEntropy/lib';
 import type { default as UnsafeEvalType } from 'internal:unsafe-eval';
 
+// TODO: export the following two functions to EmscriptenApi
```

Comment:
Easiest way would be to import them in `emscriptenSetup.ts` and assign them to `emscriptenModule` at the end of `instantiateEmscriptenModule`.